specs/latest/2.0/index.html

```diff
@@ -415,6 +415,7 @@ <h2 class="no-toc">Table of contents</h2>
 typedef (Int32Array or sequence&lt;GLint&gt;) ClearBufferIVSource;
 typedef (Uint32Array or sequence&lt;GLuint&gt;) ClearBufferUIVSource;
 typedef (Float32Array or sequence&lt;GLfloat&gt;) ClearBufferFVSource;
+typedef (ArrayBuffer or SharedArrayBuffer) ByteBuffer;
```

Comment:
Not all browser support SharedArrayBuffer yet.  I know that conformance tests are not part of this PR. When we do write them, we should make SharedArrayBuffer support be optional.